### PR TITLE
[MRG+1] PEP8-ize scrapy/commands/

### PR DESCRIPTION
**Pull request by kedark3**
See #2144
Changing multiple imports on one line to one line one import scheme. removed unused import from check.py. This is my first Pull request on Github and I am just starting my journey in open source world. Let me know if I did anything wrong in this commit.

### CODE REVIEW

File: scrapy/commands/check.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 from __future__ import print_function
 import time
-import sys
```
Comment (kmike) [inline scrapy/commands/check.py]: 👍

File: scrapy/commands/edit.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import sys, os
+import sys
+import os
```
Comment (kmike) [inline scrapy/commands/edit.py]: 👍

File: scrapy/commands/fetch.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function
-import sys, six
+import sys
+import six
```
Comment (kmike) [inline scrapy/commands/fetch.py]: 👍

File: scrapy/commands/parse.py [hunk 4]
```diff
@@ -8,7 +8,8 @@
 from scrapy.item import BaseItem
 from scrapy.utils import display
 from scrapy.utils.conf import arglist_to_dict
-from scrapy.utils.spider import iterate_spider_output, spidercls_for_request
+from scrapy.utils.spider import iterate_spider_output
+from scrapy.utils.spider import spidercls_for_request
```
Comment (kmike) [inline scrapy/commands/parse.py]: Could you please keep these imports as-is? They are perfectly fine according to pep8 ("It's okay to say this though: from subprocess import Popen, PIPE").

Comment (kedark3) [inline scrapy/commands/parse.py]: Hey @kmike I understand what you mean there. I will update it. Thank you.

**Comment by kmike** [comment]
Looks good to me.

File: scrapy/commands/startproject.py [hunk 5]
```diff
@@ -4,11 +4,15 @@
...
+from shutil import ignore_patterns
+from shutil import move
+from shutil import copy2
+from shutil import copystat
 
 import scrapy
 from scrapy.commands import ScrapyCommand
-from scrapy.utils.template import render_templatefile, string_camelcase
+from scrapy.utils.template import render_templatefile
+from scrapy.utils.template import string_camelcase
```
Comment (dangra) [inline scrapy/commands/startproject.py]: what part of pep8 says that `from ... import ...` can't reference multiple imports?

**Pull request closed without merging**